### PR TITLE
add inc/dec label size

### PR DESCRIPTION
**Pull request by tltk90**

### CODE REVIEW

File: src/app/@ansyn/imagery/model/visualizers/visualizer-style.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ export interface IFill {
 }
 
 export interface ILabel extends IStroke {
-	font?: string,
+	font?: number | any;
```
Comment (asafMasa) [inline src/app/@ansyn/imagery/model/visualizers/visualizer-style.ts]: fontSize

File: src/app/@ansyn/ol/plugins/annotations/annotations-context-menu/annotations-context-menu.module.ts [hunk 2]
```diff
@@ -6,15 +6,17 @@ import { AnnotationsWeightComponent } from './components/annotations-weight/anno
 import { AnnotationsColorComponent } from './components/annotations-color/annotations-color.component';
 import { ColorPickerComponent } from './components/color-picker/color-picker.component';
 import { ColorPickerModule } from 'ngx-color-picker';
-import { MatInputModule } from '@angular/material';
+import { MatInputModule, MatSelectModule } from '@angular/material';
```
Comment (asafMasa) [inline src/app/@ansyn/ol/plugins/annotations/annotations-context-menu/annotations-context-menu.module.ts]: MatSelectModule  - > check if we have something working in our app with our style

File: src/app/@ansyn/ol/plugins/annotations/annotations-context-menu/components/annotation-context-menu/annotation-context-menu.component.html [hunk 3]
```diff
@@ -28,22 +28,11 @@
...
-                                   #labelInput
-								   autofocus
-								   [placeholder]="'Label' | translate"
-								   name="EditLabel"
-								   [ngModel]="getFeatureProps(featureId)?.label"
-								   (ngModelChange)="updateLabel($event, featureId)">
-						</mat-form-field>
-
-					</form>
+					<ansyn-annotation-label *ngIf="selectedTab[featureId] === Tabs.Label"
```
Comment (asafMasa) [inline src/app/@ansyn/ol/plugins/annotations/annotations-context-menu/components/annotation-context-menu/annotation-context-menu.component.html]: 👍

File: src/app/@ansyn/ol/plugins/annotations/annotations-context-menu/annotations-context-menu.module.ts (hunk 2 above)
Comment (tltk90) [reply src/app/@ansyn/ol/plugins/annotations/annotations-context-menu/annotations-context-menu.module.ts]: can't use combo-box from @ansyn/ansyn inside @ansyn/ol

**Pull request merged**